less/index.less

```diff
@@ -54,6 +59,150 @@ table {
 }
 
```

Comment:
The code looks clean! My only suggestion is to make it mobile friendly (max-width 500px).